modules/installation-disk-partitioning-upi-templates.adoc

```diff
@@ -35,6 +35,54 @@ Because `/var` must be in place before a fresh installation of {op-system-first}
 If you follow the steps to create a separate `/var` partition in this procedure, it is not necessary to create the Kubernetes manifest and Ignition config files again as described later in this section.
 ====
 
+.Prerequisites
+
+* If container storage is on the root partition:
```

Comment:
Other two pre-reqs have "If the container storage", so add that to this bullet as well